app/helpers/likes_helper.rb

```diff
@@ -0,0 +1,2 @@
+# module LikesHelper
+# end
```

Comment:
- Please uncomment this code( not delete), it gives an error in your project, please fix this to have a working project 😄 

Your project:
![sgagsr](https://github.com/Cabraham1/Blog/assets/73607512/9e404399-a09c-4ce7-91b9-8e2702fe5af8)